libs/src/sdk/services/DiscoveryNodeSelector/DiscoveryNodeSelector.ts

```diff
@@ -29,11 +30,11 @@ import { AbortController as AbortControllerPolyfill } from 'node-abort-controlle
 import { mergeConfigWithDefaults } from '../../utils/mergeConfigs'
 
 const getPathFromUrl = (url: string) => {
-  const pathRegex = /^([a-z]+:\/\/)?(?:www\.)?([^\/]+)?(.*)$/
+  const pathRegex = /^([a-z]+:\/\/)?(?:www\.)?([^/]+)?(.*)$/
```

Comment:
is not escaping the forward slash in `([^/]+)` fine because of the `^`?

Reply:
that's what some google searches seemed to say